docs/Users_Guide/matthewjetstream.rst

```diff
@@ -192,68 +211,101 @@ When the installation is complete, you can verify that the Docker command line t
+
+If the command seems to succeed, confirm that the daemon is running using the status command above.
+Repeat these efforts as necessary until it is started.
 
-Once all of that is in order, you must pull the latest version of the I-WRF image onto your instance::
+Get the WRF and METplus Docker Images and the Observed Weather Data
+-------------------------------------------------------------------
 
-    docker pull ncar/iwrf
+Once Docker is running, you must pull the correct versions of the WRF and METplus images onto your instance::
+
+    docker pull ${WRF_IMAGE}
+    docker pull ${METPLUS_IMAGE}
+
+The METplus analysis will be comparing the results of the WRF simulation against
+the actual weather data that was recorded during Hurricane Matthew.
+We download that data by pulling a Docker volume that holds it,
+and then referencing that volume when we run the METplus Docker container.
+The commands to pull and create the volume are::
+
+    docker pull ncar/iwrf:data-matthew-input-obs.docker
+    docker create --name ${OBS_DATA_VOL} ncar/iwrf:data-matthew-input-obs.docker
 
 Get the Geographic Data
 -----------------------
 
-To run I-WRF on the Hurricane Matthew data set, you need a copy of the
+To run WRF on the Hurricane Matthew data set, you need a copy of the
 geographic data representing the terrain in the area of the simulation.
 These commands download an archive file containing that data,
-uncompress the archive into a folder named "WPS_GEOG", and delete the archive file.
+uncompress the archive into a folder named "WPS_GEOG" in your home directory, and delete the archive file.
 They take several minutes to complete::
 
     wget https://www2.mmm.ucar.edu/wrf/src/wps_files/geog_high_res_mandatory.tar.gz
     tar -xzf geog_high_res_mandatory.tar.gz
     rm geog_high_res_mandatory.tar.gz
 
-Create the Run Folder
----------------------
+Create the WRF and METplus Run Folders
+--------------------------------------
 
 The simulation is performed using a script that must first be downloaded.
 The script expects to run in a folder where it can download data files and create result files.
-The instructions in this exercise create that folder in the user's home directory and name it "matthew".
+The instructions in this exercise create a folder (named "wrf") under the user's home directory,
+and a sub-folder within "wrf" to hold the output of this simulation.
+The subfolder is named "20161006_00", which is the beginning date and time of the simulatition.
 The simulation script is called "run.sh".
-The following commands create the empty folder and download the script into it,
-then change its permissions so it can be run::
+Similarly, a run folder named "metplus" must be created for the METplus process to use.
+The following commands create the empty folders and download the script 
+and change its permissions so it can be run::
+
+    mkdir -p ${WRF_DIR}
+    curl --location https://bit.ly/3xzm9z6 > ${WRF_DIR}/run.sh
```

Comment:
I recommend committing this run script to the i-wrf GitHub repository so it could be copied from there instead of downloading it. This would allow us to make any necessary updates to the script more easily and keep everything that is needed contained in the repo.

Reply:
Another benefit of committing it to GitHub is you can set the file permissions of the file in GitHub so you wouldn't need to open execute permissions by hand.

Reply:
I like this idea.  I hadn't been familiar with the contents of that repo, but see the structure now, and since it is already being pulled in this document, it would simplify things a lot.  What do you think about also including the script to install Docker in that repo?  Both would need to have names that accurately and fully define their scope.

Reply:
I hesitate to put a script in the GitHub repo that is very-specific to a single environment. This script wouldn't be used by users running on another machine besides Jetstream2, so I think it would be misleading to include it in the repo. We could poll the rest of the team to see what they think.

It looks like the script is relatively simple, so the commands could be included in the instructions instead of downloading and running a script from a short URL. This may appear more transparent to users.

I am going to be writing the "Getting Started" chapter of the I-WRF User's Guide. I was planning on including basic information on how to obtain the tools needed to run, e.g. Docker or Apptainer. I was going to include information relating to specific environments that we want to support, e.g. run `module load apptainer` on Casper/Derecho to load Apptainer. Your instructions to install Docker on Jetstream2 could be included in the "Getting Started" chapter since they are not specific to the Hurricane Matthew use case and a link from your instructions could point to the Getting Started section relating to Jetstream2. What do you think? If this sounds good, I can migrate your installation instructions to the Getting Started page when I create it.